docs/setup/betamocks.md

```diff
@@ -83,15 +83,15 @@ all subsequent requests will use the cache (feel free to turn off your VA VPNs).
 You can record an error response or edit one manually to return an error status, or you can turn errors on and off
 by adding an error key to the config with an optional body. Restart rails after updating the service config:
 ```yaml
-- :method: :get
-  :path: "/wss-pciu-services-web/rest/pciuServices/v1/states"
-  :file_path: "evss/pciu_address"
+- :method: :post
+  :path: "/profile-service/profile/v3/*/*"
+  :file_path: "vet360/profile-service/default"
   :error: 400
 ```
 ```yaml
-- :method: :get
-  :path: "/wss-pciu-services-web/rest/pciuServices/v1/states"
-  :file_path: "evss/pciu_address"
+- :method: :post
+  :path: "/profile-service/profile/v3/*/*"
+  :file_path: "vet360/profile-service/default"
```

Comment:
Is vet360/profile-service/default a valid path? Where did these paths come from?

Reply:
I've changed the example to something else from [services_config](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/betamocks/services_config.yml).